src/mpid/ch4/include/mpidpre.h

```diff
@@ -177,7 +185,10 @@ typedef struct MPIDIG_req_ext_t {
         MPIDIG_acc_req_t areq;
     };
```

Comment:
I guess this is another anonymous usage you were referring. Might as well fix it?

Reply:
That one would be extensive. Probably a separate PR.